src/include/parser/postgresparser.h

```diff
@@ -172,7 +172,7 @@ class PostgresParser {
 
   // INSERT helpers
   static std::unique_ptr<std::vector<std::string>> ColumnNameTransform(List *root);
-  static std::unique_ptr<std::vector<std::vector<std::shared_ptr<AbstractExpression>>>> ValueListsTransform(List *root);
+  static std::unique_ptr<std::vector<std::vector<AbstractExpression *>>> ValueListsTransform(List *root);
```

Comment:
Why are we returning a unique vector (which is itself a pointer to an array) rather than returning the vector?

Reply:
Similarly, responded above.